client/src/components/Sidebar/menu/SubMenuCloseButton.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createPortal } from 'react-dom';
+
+import Icon from '../../Icon/Icon';
+import { gettext } from '../../../utils/gettext';
+
+export default function SubMenuCloseButton({isVisible, dispatch}) {
+  if (!isVisible) {
+    return null;
+  }
+  return createPortal(
+    <button
+      type="button"
+      onClick={
+        () => dispatch({
+          type: 'set-navigation-path',
+          path: '',
+        })
+      }
+      className="button sidebar-close-menu-button"
+      aria-label={gettext('Close submenu')}
```

Comment:
Thibaud raised that the label should be simpler, e.g. "Close" so that voice control users don't have to guess the exact label for the button. This may not be an issue for modern voice control tools, but the built-in voice control tools on macOS has this issue.